### ISSUE TITLE
A suggestion to improve the scrolling appearance of jet smart filters plugin, please check.

### ISSUE THREAD

**Issue by 79ho3ein** [CrocoBlock/suggestions#3314]
Scrolling multiple filters is the most important part of filters. But unfortunately, the appearance of the scroll bar of this plugin does not have a good user interface. https://prnt.sc/137vg3r
It would be great if you could add the ability to customize the preview bar to all the widgets in this plugin (except for the price range filter and search, of course).
In this case, we can display a more attractive scroll bar for users, an example of which you can see in the image Down.
https://prnt.sc/137vb49

Please check this feature. Thanks

**Comment by Lonsdale201** [comment]
https://www.w3schools.com/howto/howto_css_custom_scrollbar.asp Just a bit of css :)

**Comment by 79ho3ein** [comment]
True, but it's better to be able to edit it directly with jet smart filters.  Because not everyone knows css.
With thanks for your attention.

**Comment by 79ho3ein** [comment]
@SargasTM please check. thanks
